### PR TITLE
updated platform version

### PR DESCRIPTION
**Pull request by csrs**
I updated the platform version for Centos 6 and Centos 7.

### CODE REVIEW

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -5,12 +5,12 @@
...
+  version: '6.9',
 }.freeze
 
 CENTOS_7 = {
   platform: 'centos',
-  version: '7.2.1511',
+  version: '7.4.1708',
 }.freeze
 
 UBUNTU_14_04 = {
```
Comment (TravisWhitehead) [inline spec/spec_helper.rb]: @ramereth Do we still need to be supporting Ubuntu 14.04?

**Comment by TravisWhitehead** [comment]
@ramereth Also, is there any reason to keep using Travis CI on this repo? It looks like it's just doing the same as what Jenkins is, but is failing since it's still using `rubocop` instead of `cookstyle`.

File: spec/spec_helper.rb (hunk 1 above)
Comment (ramereth) [reply spec/spec_helper.rb]: Yes, and we should technically add support for the newer versions but I don't think that @csrs should do that in this PR.

**Comment by ramereth** [comment]
@TravisWhitehead I removed that webhook so it shouldn't be doing that anymore I think

@csrs please do ``git commit --amend``, just save as-is, and then do ``git push -ff`` which will trigger the checks again.

**Pull request merged**